reana_cluster/configurations/reana-cluster-minikube-dev.yaml

```diff
@@ -37,11 +40,25 @@ components:
         path: "/code/reana-server:/code"
     environment:
       - <<: *db_base_config
-      - SHARED_VOLUME_PATH: "/var/reana"
+      # Development configuration
+      # Hack to not verify SSL connections https://stackoverflow.com/questions/48391750/disable-python-requests-ssl-validation-for-an-imported-module
+      # This is useful when using a self hosted GitLab application locally
+      - CURL_CA_BUNDLE: ""
+      # Disabel CORS in development environment, for example
+      # to connect from an external React application.
+      - INVENIO_CORS_SEND_WILDCARD: "False"
+      - INVENIO_CORS_SUPPORTS_CREDENTIALS: "True"
       - WDB_SOCKET_SERVER: "wdb"
       - WDB_NO_BROWSER_AUTO_OPEN: "True"
+      # REANA configuration
+      - SHARED_VOLUME_PATH: "/var/reana"
       - FLASK_ENV:  "development"
-
+      - CERN_CONSUMER_KEY: "<CHANGEME>"
```

Comment:
`CERN_CONSUMER_KEY` is a bit confusing. Would be great to have a comment that it points to `OAuth Clients -> client_id `